### ISSUE TITLE
Demonstrate informal examples somewhere.

### ISSUE THREAD

**Issue by Raynos** [dslomov/typed-objects-es7#12]
It would be useful for people that are not familiar with specifications to see informal examples of typed objects to get a reasonable understanding of what they and what problem domain they solve.

**Comment by dslomov** [comment]
We have informal introduction here: http://wiki.ecmascript.org/doku.php?id=harmony:typed_objects.
Does it fit the bill?

**Comment by domenic** [comment]
Is it up to date? Anything on the wiki makes me automatically suspicious :).

In any case it should ideally be moved here to an Explainer.md or similar.

**Comment by dslomov** [comment]
Yes, I try to keep it up-to-date. We need to have something on a wiki, so I prefer the wiki to be an explainer and this be the spec.

**Comment by domenic** [comment]
IMO the wiki is a lost cause and nobody trusts it anymore. But if you want to try to save its reputation then up to you. _shrug_

**Comment by Raynos** [comment]
In that case add a "for informal ex platoon see wiki (link)" section to the readme :)

**Comment by bterlson** [comment]
I'd prefer to not use the wiki at all as it has no credibility...

I am hoping @wycats's guidelines for ECMAScript proposals will have a template that includes adding motivating use cases to Readme.md. Does that seem problematic?

**Comment by dslomov** [comment]
No it does not. If that is what we all do, I'll move it here (in this repo, Readme.md is the spec itself, but that is easily changable)

**Comment by domenic** [comment]
This might not be the best place to discuss the finer points of @wycats's proposal proposal but IMO the readme-as-spec works really well and a separate Explainer.md helps keep things separate.

**Comment by bterlson** [comment]
Yes, good point Domenic, the correct place to discuss this is here: https://github.com/tc39/ecma262/issues/3. (I like your proposal, though!)

**Comment by peter-lyons-kehl** [comment]
Thanks Dmitry @dslomov.

Wiki is down. Those that need an example, see http://smallcultfollowing.com/babysteps/pubs/2014.04.01-TypedObjects.pdf and https://github.com/dslomov/structs.js.
